stories/Button.stories.js

```diff
@@ -5,20 +5,21 @@ export default {
     title: 'Legacy/Button'
 };
 
-export const CtaBtn = () => '<div class="cta-btn">Leave waitlist</div>';
+const ButtonTemplate = (classes,innerHtml) => `<div class="${classes}">${innerHtml}</div>`;
```

Comment:
Would it be easier to add controls later if we pass arguments that could be used as controls?  Maybe something like this:
```suggestion
const ButtonTemplate = (buttonType, text, badgeCount=null) => `<div class="cta-btn ${ButtonTypes[buttonType]}">${text} ${badgeCount ? BadgeTemplate(badgeCount) : ''}</div>`;

const BadgeTemplate = (badgeCount) => `<span class="cta-btn__badge">${badgeCount}</span>`

const ButtonTypes = {
    default: '',
    unavailable: ' cta-btn--unavailable',
    available: ' cta-btn--available',
    preview: ' cta-btn--shell cta-btn--preview'
}
```